examples/spatialCellExample.py

```diff
@@ -32,6 +32,8 @@
    >>> import SpatialCellExample; SpatialCellExample.run()
 """
 from __future__ import print_function
+from __future__ import division
+from builtins import range
 
 import os
```

Comment:
Import ordering and spacing doesn't seem very consistent. Not a big problem. But it might be nice to cleanup some day.